python/ray/rllib/optimizers/aso_minibatch_buffer.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, inqueue, size, num_passes):
         self.inqueue = inqueue
         self.size = size
         self.max_ttl = num_passes
-        self.cur_max_ttl = 1  # ramp up slowly to better mix the input data
+        self.cur_max_ttl = num_passes  # ramp up slowly to better mix the input data
```

Comment:
is this intentional?

Reply:
Yes, the performance is better when it starts at num_passes

Reply:
I think we shouldn't change this as it has implications with other algorithms, One option is to make this an exposed variable (init_num_passes)